packages/cozy-harvest-lib/src/components/cards/LaunchTriggerCard.jsx

```diff
@@ -5,31 +5,74 @@ import Button from 'cozy-ui/transpiled/react/Button'
 import Card from 'cozy-ui/transpiled/react/Card'
 import { translate } from 'cozy-ui/transpiled/react/I18n'
 import Icon from 'cozy-ui/transpiled/react/Icon'
-import { Text } from 'cozy-ui/transpiled/react/Text'
+import { Uppercase, Text } from 'cozy-ui/transpiled/react/Text'
 
+import * as triggers from '../../helpers/triggers'
 import TriggerLauncher from '../TriggerLauncher'
 
 export class LaunchTriggerCard extends PureComponent {
   render() {
-    const { className, trigger, t, ...rest } = this.props
+    const { className, f, trigger, t, ...rest } = this.props
     return (
       <Card className={className} {...rest}>
         <TriggerLauncher trigger={trigger}>
-          {({ error, launch, running }) => (
-            <div>
-              {error && (
-                <Text className="u-error">{t('card.launchTrigger.error')}</Text>
-              )}
-              <Button
-                label={t('card.launchTrigger.button.label')}
-                icon={<Icon focusable="false" icon="sync" spin={running} />}
-                className="u-mv-half u-mh-0"
-                disabled={running}
-                onClick={launch}
-                subtle
-              />
-            </div>
-          )}
+          {({ error, launch, running, trigger }) => {
+            const lastSuccessDate = triggers.getLastSuccessDate(trigger)
+            return (
+              <div className="u-flex-medium u-flex-row-medium">
```

Comment:
```suggestion
              <div className="u-flex-m u-flex-row-m">
```
I'm gonna add this to the documentation 😅

Reply:
Thanks ! <3